### ISSUE TITLE
Minimum number of samples

### ISSUE THREAD

**Issue by ldv1** [arvkevi/kneed#43]
The following

```
from kneed import KneeLocator
import numpy as np
import matplotlib.pyplot as plt

x = [ 1, 2, 3, 4, 5, 6 ]
y = [ -481., 783., 1019., 1158., 1224., 1293 ]

kneedle = KneeLocator(x, y, S=1.0, curve='concave', direction='increasing')
kneedle.plot_knee_normalized()
plt.show()
```

gives the right answer: 
![knee](https://user-images.githubusercontent.com/12841688/64404821-78f01480-d07d-11e9-9b49-847c781e1e4f.png)

However, getting samples is time-consuming, so that I would like to come to the same conclusion with fewer samples. With x[:5] and y[:5], the answer is still correct, but with x[:4] and y[:4], no elbow can be found.
So my question is: What is the minimum number of samples required for the detection? Is there a way to make it work with only 4 samples?

Thanks !

**Comment by arvkevi** [comment]
Interesting question @ldv1 -- which of the two are you asking about?
1. Implementing early stopping, and returning the first knee point detected.
or 
2. Only giving kneed a subset of the entire system x[:4], y[:4].

I'll look into each over the weekend anyways, thanks!

**Comment by ldv1** [comment]
I was in fact asking for 1. Thanks for your support !

**Comment by arvkevi** [comment]
Hi @ldv1 it seems that your request is implemented in #50 
You can stop early by passing `online=False` to the `KneeLocator` object.
